src/App.js

```diff
@@ -9,9 +14,27 @@ class App extends Component {
 
   render() {
     return (
-      <div>
-        <Calculator />
-      </div>
+      <>
+        <nav className="nav-container">
+          <Navbar />
+          <ul className="main-nav">
+            <li>
+              <NavLink to="/">Home |</NavLink>
+            </li>
+            <li>
+              <NavLink to="/calculator">Calculator |</NavLink>
+            </li>
+            <li>
+              <NavLink to="/quote">Quote</NavLink>
+            </li>
+          </ul>
+        </nav>
```

Comment:
- **[OPTIONAL]** I think it would have been better to move this navigation to its own component.

Reply:
Thank you. I think that is a very good suggestion thanks.